charts/ks-devops/charts/jenkins/README.md

```diff
@@ -94,7 +94,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `Agent.Privileged`          | Agent privileged container                                                                         | `false`                                                                      |
 | `Agent.resources`           | Resources allocation (Requests and Limits)                                                         | `{requests: {cpu: 200m, memory: 256Mi}, limits: {cpu: 200m, memory: 256Mi}}` |
 | `Agent.volumes`             | Additional volumes                                                                                 | `nil`                                                                        |
-| `Agent.Builder.Registry`    | Agent builder image registry with namespace. Like `ghcr.io/kubesphere` or `[docker.io/]kubesphere` | `kubespheredev`
+| `Agent.Builder.Registry`    | Agent builder image registry with namespace. Like `ghcr.io/kubesphere` or `docker.io/kubesphere` | `docker.io/kubespheredev`
```

Comment:
How about my suggestion about the word like? IMO it's not accurate.

Reply:
IMO, they are examples for users. Think about if users have hosted a private registry for ks-devops. How about I change to the following?

```
Agent builder image registry with namespace. Example: {docker_registry}/{docker_id}. Available values: `ghcr.io/kubesphere` or `docker.io/kubesphere`.
```

Reply:
Another question here: Should we use `kubesphere` or `kubespheredev` as the docker ID here?

Reply:
> Another question here: Should we use `kubesphere` or `kubespheredev` as the docker ID here?

I prefer to don't change too many things here. You just want to support multiple registries here. So please just keep doing one thing.

`{docker_registry}/{docker_id}` is incorrect.  Docker is container runtime instead of equal to container runtime. There are many other container runtimes, for example: `containerd`.

I know you want to provide an example. But the following values is not an example but a real and available value:
* `ghcr.io/kubesphere`

Reply:
Make sense to me.